src/sidebar/components/Settings/General/SettingItems/UseCustomPromptsSetting/index.tsx

```
@@ -0,0 +1,200 @@
+
+  const [isButtonsDisabled, setIsButtonsDisabled] = React.useState<boolean>(true)
+  const [validationMessage, setValidationMessage] = React.useState<string>('')
+  const [isUseCustomPromptsSwitch, setIsUseCustomPromptsSwitch] = React.useState<boolean>()
+
+  const promptSchemaWithContext = React.useMemo(
+    () =>
+      z
+        .string()
+        .min(1, { message: t('settings.general.isUsingDefaultPrompt.errors.promptEmpty') })
+        .regex(
+          /(<context>[^]*<\/context>[^]*<message>[^]*<\/message>)|(<message>[^]*<\/message>[^]*<context>[^]*<\/context>)/,
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)